kolibri/content/serializers.py

```diff
@@ -0,0 +1,20 @@
+from models import ContentMetadata, File, Format
+from rest_framework import serializers
+
+
+class ContentMetadataSerializer(serializers.ModelSerializer):
```

Comment:
I don't think serializer is database specific, it only needs to know the schema in order to properly serialize the queryset.